ubuntu-25-04-vbox-amd64.pkr.hcl

```
@@ -121,13 +121,13 @@ source "virtualbox-iso" "ubuntu-25-04" {
     ["modifyvm", "{{ .Name }}", "--boot1", "dvd"],
     ["modifyvm", "{{ .Name }}", "--boot2", "disk"],
     ["modifyvm", "{{ .Name }}", "--usb", "off"],
-    ["modifyvm", "{{ .Name }}", "--vram", "64"],
+    ["modifyvm", "{{ .Name }}", "--vram", "128"],
     ["modifyvm", "{{ .Name }}", "--graphicscontroller", "vmsvga"],
-    ["modifyvm", "{{ .Name }}", "--accelerate3d", "off"],
+    ["modifyvm", "{{ .Name }}", "--accelerate3d", "on"],
     ["modifyvm", "{{ .Name }}", "--vrde", "off"],
     ["modifyvm", "{{ .Name }}", "--nic1", "nat"],
     ["modifyvm", "{{ .Name }}", "--nic2", "hostonly"],
-    ["modifyvm", "{{ .Name }}", "--hostonlyadapter2", "vboxnet0"],
+    ["modifyvm", "{{ .Name }}", "--hostonlyadapter2", "VirtualBox Host-Only Ethernet Adapter"],
```

If we change the name of the adapter to the windows version we won't be able to build the vm on linux.

Also from personal experience (tested now and last year when I had uso) the name change could be unnecessary for the following reason:
- after importing the .ova and clicking on Settings, vbox renames the adapter to the appropriate name (```VirtualBox Host-Only Ethernet Adapter``` in the case of windows), requiring the user only to save that change.

```suggestion
    ["modifyvm", "{{ .Name }}", "--hostonlyadapter2", "vboxnet0"],
```

---

I expressed myself incorrectly I think. I meant running packer through the makefile fails (at least on my linux machine) because when trying to start the vm, because of the virtualbox network adapter name (and as a result ansible playbook doesn't run).